react/components/ProductDescription/README.md

```diff
@@ -1,24 +1,44 @@
 # Product Description
-Product Description is a canonical component that any VTEX app can import.
 
-To import it into your code: 
-```js
-import ProductDescription from 'vtex.store-components/ProductDescription'
-```
+## Description
+
+`ProductDescription` is a VTEX Component that shows the description of the product.
+This Component can be imported and used by any VTEX App.
+
+:loudspeaker: **Disclaimer:** Don't fork this project, use, contribute, or open issue with your feature request.
```

Comment:
Shouldn't this be only in the main `README` of `store-components`?

Reply:
There is no problem in also putting it here IMO